spec/unit/server_spec.rb

```diff
@@ -191,6 +191,55 @@ def app
         end
       end
     end
+
+    context "broken description" do
+      before do
+        store_raw_description("foo", <<-EOT
+        {
```

Comment:
hey @fschueller I've made the test description much smaller :)

but if you know how can I create a broken description using the factory, just let me know!

thanks!